File: Praktikum 4. Loop/Soal 1/PRAK401-2310817310001-AvantioFierzaPatria.py
bilangan_kelipatan, simbol = input().split()
bilangan_kelipatan = int(bilangan_kelipatan)

output = []
for i in range(1, 51):
    if i % bilangan_kelipatan == 0:
        output.append(simbol)
    else:
        output.append(str(i))

for i in range(0, 50, 20):
    print(" ".join(output[i:i+20]))

File: Praktikum 4. Loop/Soal 1/PRAK401-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

int main()
{
    int bilangan_kelipatan;
    char simbol;
    scanf("%d %c", &bilangan_kelipatan, &simbol);

    int i;
    for (i = 1; i <= 50; i++)
    {
        if (i % bilangan_kelipatan == 0)
        {
            printf("%c ", simbol);
        }
        else
        {
            printf("%d ", i);
        }

        if (i % 20 == 0)
        {
            printf("\n");
        }
    }

    return 0;
}
